config.json

```diff
@@ -450,6 +450,19 @@
         ],
         "difficulty": 2
       },
+      {
+        "slug": "resistor-color-duo",
+        "name": "Resistor Color Duo",
+        "uuid": "a6b5f8e0-96a5-4da1-97ad-033a2a4b115b",
+        "practices": [
+          "lists"
```

Comment:
Let's add pattern matching.
Let's add pattern matching and custom types to prerequisites as well.